src/test/java/StringTest.java

```diff
@@ -0,0 +1,56 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+    @Test
+    void split_bulk() {
+        String str = "1,2";
+
+        String[] result = str.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?